app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/offer/QuoteCard.kt

```diff
@@ -127,9 +127,9 @@ private fun QuoteDetailsAndPrice(isExpanded: Boolean, quote: MoveQuote) {
         Spacer(Modifier.width(8.dp))
         val angle = animateFloatAsState(
           targetValue = if (isExpanded) {
-            0f
-          } else {
             -180f
+          } else {
+            0f
           },
```

Comment:
Was it flipped the wrong way? 😅

I think we want this to point down when the content is shrinked, and to point up when it's expanded. Sort of to have the arrow point towards where the content will go when clicked.

Reply:
Yep, was flipped and felt strange:)

Reply:
I was also thinking of adding a gigantic ripple that covers all the card there (we had it in payments somewhere - smaller round ripple in the center of card looking a bit strange), but it's a smaller thing